Sources/BrowserServicesKit/PrivacyConfig/AppPrivacyConfiguration.swift

```diff
@@ -46,10 +46,37 @@ public struct AppPrivacyConfiguration: PrivacyConfiguration {
         return data.trackerAllowlist.state == PrivacyConfigurationData.State.enabled ? data.trackerAllowlist.entries : [:]
     }
     
-    public func isEnabled(featureKey: PrivacyFeature) -> Bool {
+    func parse(versionString: String) -> [Int] {
+        return versionString.split(separator: ".").map { Int($0) ?? 0 }
+    }
+    
+    func staisfiesMinVersion(feature: PrivacyConfigurationData.PrivacyFeature,
+                             versionProvider: AppVersionProvider) -> Bool {
+        if let minSupportedVersion = feature.minSupportedVersion,
+           let appVersion = versionProvider.appVersion() {
+            let minVersion = parse(versionString: minSupportedVersion)
+            let currentVersion = parse(versionString: appVersion)
+            if minVersion.count != currentVersion.count {
+                // Config error
+                return false
+            }
+            
+            for i in 0..<minVersion.count {
+                if minVersion[i] > currentVersion[i] {
+                    return false
+                }
+            }
```

Comment:
Should this also return true if a component of the current version is greater than the minimum version? 

* Min version `0.22.3` and current version `1.0.0` fails as the second component 22 > 0, but the first component of the current version is greater so it should take precedence
* Min version `0.22.3` and current version `0.23.0` fails as the third component 3 > 0